plugins/q/src/formatter/parts/qs-number-formatter.js

```diff
@@ -145,6 +145,15 @@ function getAbbreviations(localeInfo, listSeparator) {
   return abbreviations;
 }
 
+function isSIAbbreviation(pattern) {
+  if (pattern.indexOf('A') === -1) {
+    return false;
+  }
+  const validFormatCodeCharacters = ['#', '0', ' ']; // The last character is a non-breaking space, not a normal space
+  const formatCode = pattern.substring(0, pattern.indexOf('A'));
+  return [...formatCode].every((c) => validFormatCodeCharacters.includes(c));
+}
```

Comment:
this need to mark all format created here: https://github.com/qlik-oss/picasso.js/blob/02620ff4f8e3c89e14539bc2387c0524599322a8/plugins/q/src/formatter/index.js#L26 as valid.
that have the format `#${decimal}##A`;

A question is if we want it to be possible for the end user to specify a format that triggers SI abbreviation
or if it should only when auto format is detected.
(if not we could replace the use of A with an extra parameter outside the pattern)

Reply:
Thanks, @T-Wizard . I will fix the code to consider the case you mentioned.